phpBB/phpbb/db/doctrine/table_helper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\doctrine;
+
+class table_helper
+{
+	/**
+	 * Converts phpBB's column representation to Doctrine's representation.
+	 *
+	 * @param array $column_data Column data.
+	 *
+	 * @return array A pair of type and array of column options.
+	 */
+	public static function convert_column_data(array $column_data): array
+	{
+		$data = [
+			'',
+			[]
+		];
+
+		return $data;
```

Comment:
Can just combine that with the above array declaration, not really needing an extra variable.

Reply:
This function is WIP.